addons/web/static/lib/odoo_ui_icons/style.css

```diff
@@ -62,3 +62,17 @@
 .oi-view:before { content: '\e861'; }
 .oi-archive:before { content: '\e862'; }
 .oi-unarchive:before { content: '\e863'; }
+
+/* RTL adaptations. */
+/* Flip directional icons by 180 degree. */
+/* ---------------------------------------------------------------------------- */
+.o_rtl .oi-chevron-left,
+.o_rtl .oi-chevron-right,
+.o_rtl .oi-arrow-down-left,
+.o_rtl .oi-arrow-down-right,
+.o_rtl .oi-arrow-left,
+.o_rtl .oi-arrow-right,
+.o_rtl .oi-arrow-up-left,
+.o_rtl .oi-arrow-up-right {
+  transform: rotate(180deg);
+}
```

Comment:
Any reason to repeat `o_rtl` on every line instead of keeping it nested as before?

Reply:
Didn't see it was a `css` file